coresdk/src/coresdk/database.h

```diff
@@ -1,60 +1,301 @@
-//
-//  database.h
-//  splashkit
-//
-//  Created by Jake Renzella on 2/08/2016.
-//  Copyright © 2016 Andrew Cain. All rights reserved.
-//
+/**
+ * @header Database
+ * @author Jake Renzella
+ * @brief SplashKit Database allows you to create, query and otherwise manipulate a database.
+ *
+ * The SplashKit's database library allows you to easily create or load databases
+ * and to perform queries on them. To get started with databases in SplashKit,
+ * the first thinkg you need to do is to load or create a database file. You can
+ * do this by calling the `open_database(string name)` function.
+ */
 
 #ifndef database_h
 #define database_h
 
 #include <string>
 using namespace std;
 
+/**
+ * The `database` type is used to refer to databases that can be
+ * manipulated by the SplashKit database code. Databases are:
+ *
+ *   - opened with `open_database`,
+ *
```

Comment:
Awesome, we need to remember that lists **must** be separated with **two** newlines (it's not the best but for now it works). Good to see! 😄